client_test.go

```diff
@@ -907,6 +907,133 @@ func TestCreatePage(t *testing.T) {
 			},
 			expError: nil,
 		},
+		{
+			name: "valid case: ParentType is ParentTypeDatabase and successful response",
```

Comment:
Could you please change the name to:
```suggestion
			name: "database parent, successful response",
```

And change the name for the other (existing) test case to "page parent, successful response"? 🙏